plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -1717,5 +1717,7 @@ private function search_filter() {
 				<option value="<?php echo esc_attr( wp_unslash( sanitize_text_field( $value ) ) ); ?>" <?php selected( $value, sanitize_text_field( wp_unslash( $selected ) ) ); ?>><?php echo esc_html( $label ); ?></option>
 				<?php
 			}
+			
+		echo '</select>';
```

Comment:
```suggestion
			<?php } ?>
		</select>
		<?php
```

This would just make the method more consistent in how it outputs content.